src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -409,7 +409,11 @@ public function testFormatWithDateTimeZoneGmtOffset()
 
     public function testFormatWithIntlTimeZone()
     {
-        if (PHP_VERSION_ID < 50500 && !(extension_loaded('intl') && method_exists('IntlDateFormatter', 'setTimeZone'))) {
+        if (!extension_loaded('intl')) {
+            $this->markTestSkipped('Int extension not installed.');
```

Comment:
@nicolas-grekas @iltar 

For this case done.

Another skip conditions in this test can't be done by requires annotation, it is not support conditions and etc.